### DISCUSSION TITLE
Welcome to KNX-Ultimate Discussions!

### DISCUSSION THREAD

**Discussion by Supergiovane** [Supergiovane/node-red-contrib-knx-ultimate#254] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sunstarfire** [comment]
Looks like KNX Secure is not working.... I cannot select Tunnel TCP, it only offers Multicast and Tunnel UDP...

Edit: Tried to switch off secure, no change. The error I get is "Something went wrong. Please create at least a KNX Gateway node first"...

**Comment by Supergiovane** [comment]
KNX secure is  not implemented . We are working on that but we doesn’t have any deadline.

**Comment by Supergiovane** [comment]
And please do a full deploy. Please also read the wiki, there are all infos you need.
